duckduckgo/content-scope-scripts
# Design feedback on update notification

**Asana Task/Github Issue:** https://app.asana.com/0/1201141132935289/1208684961069930/f
## Description

- [x] Design feedback applied

## Testing Steps

- visit: https://deploy-preview-1190--content-scope-scripts.netlify.app/build/pages/new-tab/?update-notification=populated
- Ensure the `dismiss` icon is always present

## Checklist


*Please tick all that apply:*
